bin/reorder-keys

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+if [ ! -d exercises ] ; then
+    echo "\"exercises\" directory not found, make sure you're running from the repository root" >&2
+    exit 1
+fi
+
+for canonical_data_file in exercises/*/canonical-data.json ; do
+    jq 'walk(if (type == "object") and has("uuid") then . as $obj | reduce ["uuid", "reimplements", "description", "comments", "scenarios", "property", "input", "expected"][] as $key ({}; . + (if $obj | has($key) then { ($key): $obj[$key] } else {} end)) else . end)' $canonical_data_file > tmp && mv tmp $canonical_data_file    
```

Comment:
I'm not great at jq, so if there is a more elegant way to do this, do let me know!